wildfly-standalone-centos7/scripts/wildfly-setup-redhat.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+echo "Red Hat WILDFLY 16.0.0.Final Standalone Intallation Start..." >> /home/$1/install.log
+/bin/date +%H:%M:%S  >> /home/$1/install.log
+
+export WILDFLY_USER=$2
+export WILDFLY_PASSWORD=$3
+
+echo "WILDFLY_USER: " ${WILDFLY_USER} >> /home/$1/install.log
+echo "WILDFLY_PASSWORD: " ${WILDFLY_PASSWORD} >> /home/$1/install.log
+
+echo "WILDFLY Downloading..." >> /home/$1/install.log
+cd /home/$1
+yum install -y git unzip java
+curl https://download.jboss.org/wildfly/16.0.0.Final/wildfly-16.0.0.Final.zip -o wildfly-16.0.0.Final.zip
```

Comment:
Add some error handling and retries to the download

Reply:
Provide the guidelines.

Reply:
@bmoore-msft Any updates?